### PR TITLE
add README documentation for routes, business logic, KID, DB tables

### PR DESCRIPTION
**Pull request by UberFarmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Effect Foundation API
+## Effect Foundation API
```
Comment (UberFarmer) [inline README.md:1]: Switched to ## so my auto-table-of-contents plugin can include this, and doesn't indent everything else

File: README.md [hunk 2]
```diff
@@ -153,21 +165,64 @@ We have three main branches in the repository, `master`, `stage` and `dev`. Any
...
+- `/referrals`: CRUD endpoints for donor referrals
+- `/paypal`: endpoints for Paypal payments
+- `/vipps`: endpoints for Vipps payments and payment agreements
+- `/avtalegiro`: endpoints for Avtalegiro payments and payment agreements
+- `/facebook`: endpoints for Facebook payments
+- `/scheduled`: endpoints called by cron jobs (usually daily), mostly for processing recurring payments
+- `/logging`: When we process recurring payments, we store some info. These endpoints fetch that info.
+- `/mail`: endpoints for sending certain emails
+- `/reports`: manual import data in bulk, via documents from banks or payment providers
+- `/csr`: ?
```
Comment (UberFarmer) [inline README.md:204]: couldn't figure out what this was an acronym for, or what the grouped functionality was here

Comment (fellmirr) [reply README.md:204]: @UberFarmer CSR is an unused endpoint which previously was used from som Corporate Social Responsibility stuff. We should probably just removed all of that code :)

**Review by fellmirr — APPROVED** [review]
@UberFarmer Stellar work!

Fantastic improvement, I think this is a very nice broad overview of the structure from an outside perspective 👍

**Comment by UberFarmer** [comment]
@fellmirr looks like I have access to merge PRs myself!

**Pull request merged**